docker-stack.yml

```diff
@@ -19,10 +19,13 @@ services:
       - '--storage.tsdb.path=/prometheus'
       - '--web.console.libraries=/usr/share/prometheus/console_libraries'
       - '--web.console.templates=/usr/share/prometheus/consoles'
+      - '--web.enable-lifecycle'
+      - '--web.enable-admin-api'
```

Comment:
Is this required for push gateway?

Reply:
no, `--web.enable-admin-api` As of Prometheus 2.0, the --web.enable-admin-api flag controls access to the administrative HTTP API which includes functionality such as deleting time series.
`--web.enable-lifecycle` Controls HTTP reloads and shutdowns of Prometheus. This is also disabled by default. If enabled they will be accessible under the /-/reload and /-/quit paths.

Reply:
OK, did investigation on this. Thanks for clarifying and sorry for the delay